src/modules/Dashboard/features/RespondentData/RespondentDataReview/EmptyResponses/EmptyResponses.styles.ts

```diff
@@ -0,0 +1,18 @@
+import { styled } from '@mui/material';
+
+import { variables, StyledFlexAllCenter } from 'shared/styles';
+
+export const StyledEmptyReview = styled(StyledFlexAllCenter)`
+  flex-direction: column;
+  width: 40rem;
+  height: 100%;
+  text-align: center;
+
+  svg {
+    fill: ${variables.palette.outline};
+  }
+`;
+
+export const StyledWrapper = styled(StyledFlexAllCenter)`
+  height: calc(100% - 9.6rem);
```

Comment:
I suppose it's a header height or similar offset, Is it possible to use a variable for it?